ListDetail/build.gradle

```diff
@@ -28,8 +28,10 @@ dependencies {
     implementation androidxDependencies.constraintLayout
     implementation androidxDependencies.ktxCore
     implementation androidxDependencies.ktxFragment
+    implementation androidxDependencies.recyclerView
 
-    implementation microsoftDependencies.dualScreenLayout
+//    implementation microsoftDependencies.dualScreenLayout
+    implementation "com.microsoft.device.dualscreen:layouts:1.0.0-alpha02"
```

Comment:
Update the dependency here, since the rest of samples haven't been updated.

Reply:
Please Joy can you add a `//TODO message` here? so we don't miss it when we change the artifacts.